website/static/js/osfToggleheight.js

```diff
@@ -0,0 +1,84 @@
+
+'use strict';
+var jQuery = require('jquery');
+
+(function ( $, window, document, undefined ) {
+
+	// Create the defaults once
+	var pluginName = 'osfToggleHeight';
+	var defaults = {
+			height:  50,
+			iconDown : 'fa fa-angle-down',
+			iconUp : 'fa fa-angle-up'
+	};
+
+	// The actual plugin constructor
+	function ToggleHeight ( element, options ) {
+		var self = this;
+		self.element = element;
+		self.$el = $(element);
+		self.settings = $.extend( {}, defaults, options );
+		self._defaults = defaults;
+		self._name = pluginName;
+
+		self.init =  function () {
+		    self.collapsed = true;
+			self.showToggle = false;
+			self.gradientDiv = $('<div class="' + self.element.id + '-gradient toggle-height-gradient" style="display:none"></div>').appendTo(self.element);
```

Comment:
It would be worth documenting in the docstring for this module that the selected element *must* have an ID, or else this will break.